scripts/mujoco/evaluate_policy.py

```diff
@@ -0,0 +1,30 @@
+import gymnasium as gym
+from huggingface_sb3 import load_from_hub
+from stable_baselines3 import SAC
+from stable_baselines3.common.evaluation import evaluate_policy
+
+ENV_ID = "Ant"
+ALGORITHM = "SAC"
+# PROFICIENCY = "medium"
+PROFICIENCY = "expert-fine-tuned"
+
+eval_env = gym.make(f"{ENV_ID}-v5")
+eval_env = gym.make(f"{ENV_ID}-v5", include_cfrc_ext_in_observation=False)
+
+match ALGORITHM:
+    case "SAC":
+        expert_model_checkpoint = load_from_hub(
+            repo_id=f"farama-minari/{ENV_ID}-v5-{ALGORITHM.upper()}-{PROFICIENCY}",
+            filename=f"{ENV_ID.lower()}-v5-{ALGORITHM.lower()}-{PROFICIENCY}.zip",
+        )
+        model = SAC.load(expert_model_checkpoint)
+    case "TD3":
+        None
+    case "PPO":
+        None
```

Comment:
why?

Also this one is removable, cool for sanity checking while you are making the dataset

Reply:
moved it https://gist.github.com/Kallinteris-Andreas/f4c5b7457517a220b9338322d51fdcf8